theoriq/dialog/tool_items.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Annotated, Literal
+
+from pydantic import Field
+
+from .block import BaseData, BlockBase
+
+
+class ToolCallData(BaseData):
+    name: Annotated[str, Field(..., description="Name of the tool to call")]
+    arguments: Annotated[str, Field(..., description="JSON string of arguments to pass to the tool")]
+    id: Annotated[str, Field(..., description="ID of the tool call")]
+
+
+class ToolCallBlock(BlockBase[ToolCallData, Literal["internal:tool_call"]]):
```

Comment:
I'd rather go with `camelCase` instead of `snake_case` for consistency with APIs
```suggestion
class ToolCallBlock(BlockBase[ToolCallData, Literal["internal:toolCall"]]):
```